src/hyperion/parameters/beamline_parameters.py

```diff
@@ -17,10 +18,11 @@ def __getitem__(self, item: str):
         return self.params[item]
 
     @classmethod
```

Comment:
Nit: Generally it's not very pythonic to have a class with lots of static methods, it's preferred to just use bare functions, in this case we're just following what was there before and it's not too bad so not a problem